proxy/envoy/testdata/envoy-v1.json.golden

```diff
@@ -439,6 +439,24 @@
         ]
       },
       {
+        "name": "mixer_server",
```

Comment:
Is mixer_server an upstream / cluster now ?
How do statically defined clusters interact with CDS ?